src/Entity/EventRegistration.php

```diff
@@ -144,4 +144,11 @@ public function matches(BaseEvent $event, Adherent $adherent = null): bool
 
         return $adherent && $this->adherentUuid->equals($adherent->getUuid());
     }
+
+    public function getLastNameInitial(): string
```

Comment:
what about using `EntityPersonNameTrait` in `EventRegistration` too for avoiding CC this method?